pom.xml

```diff
@@ -28,7 +28,7 @@
         <jdk.version>1.8</jdk.version>
 
         <!-- Maven and Surefire plugins -->
-        <maven-compiler-plugin.version>3.1</maven-compiler-plugin.version>
+        <maven-compiler-plugin.version>3.5.1</maven-compiler-plugin.version>
```

Comment:
shouldnt this be `<maven-compiler-plugin.version>3.1</maven-compiler-plugin.version>`?

Reply:
3.1 does not allow the use of annotationProcessorPaths so required 3.5.1